quirks/password-rules.json

```diff
@@ -377,6 +377,9 @@
     "splunk.com": {
         "password-rules": "minlength: 8; maxlength: 64; required: lower; required: upper; required: digit; required: [-!@#$%&*_+=<>];"
     },
+    "store.nvidia.com": {
```

Comment:
Also, quick clarification here. Our README states:

```
When a domain is listed in quirks/password-rules.json, it means that that domain and all of its subdomains use the rule.
```

I'm assuming this means that when specifying `store.nvidia.com` the rule is allowed to match `*.store.nvidia.com`, as opposed to `*.nvidia.com`, and if true I'll open a PR to make that explicitly clear in the README.

Reply:
That’s exactly right, @Cldfire!